### PR TITLE
task/Introduction-React : Completed Task & Summary

### PR DESCRIPTION
**Pull request by Skmnkn**
Task and Summary Introduction React

Link Soal [Introduction React](https://docs.google.com/document/d/1zd9zBTEOPISZIYRQl5BmcZDdoZ-8VPHKVE_Hn3qm10w/edit)

Link [Resource Task React](https://drive.google.com/drive/folders/1Nw2lg57SE3sKkhD5QAO9rhvfsE6ELR_X)

- [x] Instalasi React cra
- [x] Membuat home.html dan ubah menjadi home,js dengan [HTML to JSX](https://magic.reactjs.net/htmltojsx.htm)
- [x] Menambahkan Screenshot home page
- [x] Menyelesaikan Summary Introduction React

### CODE REVIEW

File: 11_Introduction-React/task_react/src/modules/about.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        <div className="container">
+          <div className="row pt-5 pb-5">
+            <div className="pl-5 col-lg-5 col-md-4 col-sm-12 col-12 text-lg-left text-md-left text-sm-center">
+              <a href="index.html">
+                <img className="logo-bawah" src={downLogo} />
+              </a>
+            </div>
+            <div className="col-lg-3 col-md-3 col-sm-12 col-12 text-lr-left text-md-center text-sm-center">
+              <h6>Social Media:</h6>
+              {/* <div class="col-3 text-center"> */}
```
Comment (syukronarie) [inline 11_Introduction-React/task_react/src/modules/about.js]: hapus sintax yg gak dipake lagi

File: 11_Introduction-React/task_react/src/modules/about.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+              <a href="https://www.facebook.com/AlterraAcademy">
+                <img className="logo-sosmed" src={facebook} />
+              </a>
+              {/* </div> */}
+              {/* <div class="col-3 text-center"> */}
```
Comment (syukronarie) [inline 11_Introduction-React/task_react/src/modules/about.js]: hapus sintax yg gak dipake lagi

File: 11_Introduction-React/task_react/src/modules/about.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+              <a href="https://twitter.com/">
+                <img className="logo-sosmed" src={twitter} />
+              </a>
+              {/* </div> */}
+              {/* <div class="col-3 text-center"> */}
```
Comment (syukronarie) [inline 11_Introduction-React/task_react/src/modules/about.js]: hapus sintax yg gak dipake lagi

File: 11_Introduction-React/task_react/src/modules/about.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+              <a href="https://www.instagram.com/alterra.academy/">
+                <img className="logo-sosmed" src={instagram} />
+              </a>
+              {/* </div> */}
+              {/* <div class="col-3 text-center"> */}
```
Comment (syukronarie) [inline 11_Introduction-React/task_react/src/modules/about.js]: hapus sintax yg gak dipake lagi

File: 11_Introduction-React/task_react/src/modules/about.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+              <a href="https://id.linkedin.com/">
+                <img className="logo-sosmed" src={linkedin} />
+              </a>
+              {/* </div> */}
```
Comment (syukronarie) [inline 11_Introduction-React/task_react/src/modules/about.js]: hapus sintax yg gak dipake lagi

File: 11_Introduction-React/task_react/src/modules/home.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+            <div className="row align-items-center">
+              <div className="col-lg-8 col-md-12 col-sm-12 col-12">
+                <div className="alterra">
+                  <a href="index.html">
+                    <img className="logo-atas" src={profileLogo} alt="logo" />
+                  </a>
+                </div>
+              </div>
+              <div className="col-lg-4 col-md-12 col-sm-12 col-12">
+                {/* <div class="container"> */}
```
Comment (syukronarie) [inline 11_Introduction-React/task_react/src/modules/home.js]: hapus sintax yg gak dipake lagi

File: 11_Introduction-React/task_react/src/modules/home.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+                    <a href="about.html">ABOUT</a>
+                  </div>
+                  <div className="col-lg-3 col-md-3 col-sm-3 text-md-center text-sm-center text-lg-left">
+                    <a href="#">EXPERIENCE</a>
+                  </div>
+                  <div className="col-lg-3 col-md-3 col-sm-3 text-md-center text-sm-center text-lg-right">
+                    <a href="form.html">CONTACT</a>
+                  </div>
+                </div>
+                {/* </div> */}
```
Comment (syukronarie) [inline 11_Introduction-React/task_react/src/modules/home.js]: hapus sintax yg gak dipake lagi

**Review by syukronarie — APPROVED** [review]

**Comment by Skmnkn** [comment]
Sudah saya delete mas unused syntax nya, thx

**Pull request merged**